lib/bolognese/writers/schema_org_writer.rb

```diff
@@ -46,6 +46,13 @@ def schema_hsh
       def schema_org
         JSON.pretty_generate schema_hsh
       end
+
+      private
+
+      def abstract_description
+        # Fetch the first description with descriptionType "Abstract"
+        descriptions&.find { |d| d["descriptionType"] == "Abstract" }
+      end
```

Comment:
i'm just wondering if we could perhaps DRY this implementation since we do repeat this method. I'm not sure about the code structure but maybe it could go into a writers concern or in some utility module? if this is not possible please do not hesitate to let me know.

Reply:
@wendelfabianchinsamy moved the method to the common place.

Reply:
Brilliant! Nice one @ashwinisukale 🚀.